src/reactivate-recurring-reminder/lib/db.ts

```diff
@@ -0,0 +1,22 @@
+import { Pool, QueryConfig, QueryResult } from 'pg';
+import { runWithLogging } from '../../lib/db';
+import { Reactivation } from '../lambda/models';
+
+export async function reactivateRecurringReminder(
+	reactivation: Reactivation,
+	pool: Pool,
+): Promise<QueryResult> {
+	const query: QueryConfig = {
+		text: `
+			UPDATE
+				recurring_reminder_signups
+			SET
+				reminder_cancelled_at = NULL
```

Comment:
This should also set the `reminder_created_at` to the current timestamp too.

Reply:
yeah! 👍